web_ir_actions_act_multi/models/ir_actions.py

```diff
@@ -5,9 +5,9 @@
 
 class IrActionsActMulti(models.Model):
     _name = "ir.actions.act_multi"
-    _description = "Action Mulit"
+    _description = "Action Multi"
     _inherit = "ir.actions.actions"
-    _table = "ir_actions"
+    _table = "ir_actions_act_multi"
```

Comment:
Yes, but this change very likely needs migration scripts, so that upgrading from Odoo 16, data is correctly separated and cleaned up from the `ir_actions` table and added to `ir_actions_act_multi` accordingly.